gammapy/makers/tests/test_safe.py

```diff
@@ -57,9 +57,17 @@ def test_safe_mask_maker(observations, caplog):
     mask_edisp_bias = safe_mask_maker.make_mask_energy_edisp_bias(dataset)
     mask_edisp_bias_offset = safe_mask_maker_offset.make_mask_energy_edisp_bias(dataset, obs)
     assert_allclose(mask_edisp_bias.data.sum(), 1815)
-    assert_allclose(mask_edisp_bias_offset.data.sum(), 121)
+    assert_allclose(mask_edisp_bias_offset.data.sum(), 1694)
```

Comment:
Why such a large difference?

Reply:
there is multiple roots in this case and previously it returned one of them but not necessarily the first one